mabwiser/base_mab.py

```diff
@@ -61,12 +61,12 @@ class BaseMAB(metaclass=abc.ABCMeta):
     """
 
     @abc.abstractmethod
-    def __init__(self, rng: np.random.RandomState, arms: List[Arm], n_jobs: int, backend: str = None):
+    def __init__(self, rng: _NumpyRNG, arms: List[Arm], n_jobs: int, backend: str = None):
```

Comment:
I think this should be _BaseRNG to allow for future changes to the library

Reply:
Good catch! It should be _BaseRNG